tests/integration/pybaseball/test_statcast_fielding.py

```diff
@@ -18,7 +18,7 @@ def test_statcast_outs_above_average() -> None:
 	assert not result.empty
 
 	assert len(result.columns) == 17
-	assert len(result) == 184
+	assert len(result) == 182
```

Comment:
Relax this assert
Read the article: "You'll never believe these 42 HIGHLY EFFECTIVE ways Tom Tango found to mess with @TheCleric"